README.md

```diff
@@ -27,14 +27,17 @@ $ gh repo-export --help
 Bulk exports a list of Git repositories from an organization
 
 USAGE
-  gh-repo-export [options] <organization> <repo> ...
+  gh-repo-export [options] <organization> <repo1> <repo2> ...
 
 FLAGS
   -d, --debug                         Enable debugging
       --exclude-attachments           Indicates attachments should be excluded from the migration
       --exclude-releases              Indicates releases should be excluded from the migration
       --exclude-owner-projects        Indicates projects owned by the organization or users should be excluded from the migration
+      --exclude-metadata              Indicates metadata should be excluded from the migration
+      --exclude-git-data              Indicates git data should be excluded from the migration
       --hostname=string               Hostname of the GitHub instance to authenticate with
+      --pat=string                    PAT to authenticate with
```

Comment:
Can you help me understand why this as a flag versus using the [GH CLI `GH_TOKEN` or `GITHUB_TOKEN` env var](https://cli.github.com/manual/gh_help_environment)?

Reply:
Oops! That was a holdover from something else I was trying and shouldn't have made it in here :)